# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/xdelta/gcc40.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xdelta-1.1.3/xdmain.c~	2001-09-24 08:59:18.000000000 +0200
+++ xdelta-1.1.3/xdmain.c	2005-10-21 02:30:09.000000000 +0200
@@ -1441,7 +1441,7 @@
   return TRUE;
 }
 
-static HandleFuncTable xd_handle_table =
+HandleFuncTable xd_handle_table =
 {
   (gssize (*) (FileHandle *fh)) xd_handle_length,
   (gssize (*) (FileHandle *fh)) xd_handle_pages,
